surveys/2024-annual-survey/translations/fr.md

```diff
@@ -2,29 +2,29 @@
 
 Type: select one
 
-- Oui, j'utilise Rust (pour tout usage, même si vous êtes en phase d'apprentissage).
-- Non, je n'utilise pas Rust actuellement, mais je l'ai fait dans le passé.
-- Non, je n'ai jamais utilisé Rust.
+- Oui, j’utilise Rust (pour tout usage, même si vous êtes en phase d'apprentissage).
+- Non, je n’utilise pas Rust actuellement, mais je l’ai fait dans le passé.
+- Non, je n’ai jamais utilisé Rust.
```

Comment:
what did it change here? The apostroph has become some other typographic symbol?

I see the same change everywhere

Reply:
The english one has been replaced by the correct one in French (at least the one for France) (it is more curly)

Reply:
It's `U+2019`, it's a "guillemet-apostrophe", and it's even recommended by Unicode it-self[^1]:

> 2019 `’` is preferred for apostrophe

[^1]: https://www.unicode.org/charts/PDF/U0000.pdf

Reply:
wow TIL :+1: